home-work7.py

```diff
@@ -0,0 +1,40 @@
+phonebook = dict()
+commands = ["stats", "add", "delete", "list", "show"]
+while True:
+    com = (input("Please choose your command: stats, add, delete <name>, list, show <name>")).split()
+    print(com)
+    if com[0] not in commands:
+        print("wrong command")
+    else:
+        match com[0]:
+            case "stats":
+                print(f"There are {len(phonebook)} numbers in your phone book ")
+
+            case "add":
+                name = input("Please enter new name")
+                number = input("Please enter phone number")
+                phonebook[name] = number
```

Comment:
The task was not to override records. So it is better to check if such record already exists before add this record

Reply:
was changed, please see in new commit